packages/envoyproxy/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,99 @@
+---
+description: Pipeline for normalizing envoyproxy logs
+processors:
+- set:
```

We should review whether we need to change the default settings of `ignore_empty_value` and `ignore_failure` from `false` to `true` for all processors in use in all pipelines.

---

This was copied from our beats pipeline. I dont think its necessary for these to be here as they are mostly for the `timestamp` field and I have yet to see this be a `null` value. Im going to remove these settings

---

Unsure if i should remove [this](https://github.com/tehbooom/integrations/blob/b42201c501cb0cd720cb08d5abeb7347bb1385c1/packages/envoyproxy/data_stream/log/elasticsearch/ingest_pipeline/json.yml#L13). I think the agent running on k8s will populate the `kubernetes` field. Would we want to overwrite that? Doesnt this `ignore_failure` handle that? I think if it does we can add a conditional to verify its not already set versus ignoring errors

---

I think ignore_failure should handle. Let's keep it for now.